src/libertem/api.py

```diff
@@ -17,42 +18,54 @@
 from libertem.analysis.point import PointMaskAnalysis
 from libertem.analysis.masks import MasksAnalysis
 from libertem.analysis.base import AnalysisResultSet, Analysis
-from libertem.udf.base import UDFRunner, UDF
+from libertem.udf.base import UDFRunner, UDF, UDFResults
 from libertem.udf.auto import AutoUDF
+from libertem.utils.async_utils import async_generator
+from libertem.viz.mpl import MPLLive2DPlot
```

Comment:
Would it be possible to import this class lazily, to keep initial import time minimal? Maybe only if plotting is used at all? Could keep `self.plot_class` as `None` by default, and only when it is initialized in `_prepare_plots`, import `MPLLive2dPlot`. What do you think?

Reply:
I've turned `plot_class` into a property, leave it `None` by default and load the default class upon access. That way external users can always access it, but it is only loaded if needed. Sounds good? :-)